ios/FluentUI/Vnext/Notification/FluentNotification.swift

```diff
@@ -42,24 +48,31 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
     /// - Parameters:
     ///   - style: `MSFNotificationStyle` enum value that defines the style of the Notification being presented.
     ///   - shouldSelfPresent: Whether the notification should  present itself (SwiftUI environment) or externally (UIKit environment)
-    ///   - message: Text for the main title area of the control. If there is a title, the message becomes subtext.
+    ///   - message: Optional text for the main title area of the control. If there is a title, the message becomes subtext.
+    ///   - attributedMessage: Optional attributed text for the main title area of the control. If there is a title, the message becomes subtext. If set, it will override the message parameter.
     ///   - isPresented: Controls whether the Notification is being presented.
     ///   - title: Optional text to draw above the message area.
+    ///   - attributedTitle: Optional attributed text to draw above the message area. If set, it will override the title parameter.
     ///   - image: Optional icon to draw at the leading edge of the control.
     ///   - actionButtonTitle:Title to display in the action button on the trailing edge of the control.
     ///   - actionButtonAction: Action to be dispatched by the action button on the trailing edge of the control.
     ///   - messageButtonAction: Action to be dispatched by tapping on the toast/bar notification.
     public init(style: MSFNotificationStyle,
                 shouldSelfPresent: Bool = true,
-                message: String,
+                message: String = "",
```

Comment:
If you're changing the message property to be optional, I think this should follow and the default value should be nil like the other ones.

Thoughts?